tests/examples/test_request_data.py

```diff
@@ -47,28 +50,30 @@ def test_request_data_5() -> None:
     with TestClient(app=app_5) as client:
         response = client.post(
             "/",
-            content=b'--d26a9a4ed2f441fba9ab42d04b42099e\r\nContent-Disposition: form-data; name="id"\r\n\r\n1\r\n--d26a9a4ed2f441fba9ab42d04b42099e\r\nContent-Disposition: form-data; name="name"\r\n\r\nJohn\r\n--d26a9a4ed2f441fba9ab42d04b42099e--\r\n',
-            headers={
-                "Content-Length": "211",
-                "Content-Type": "multipart/form-data; boundary=d26a9a4ed2f441fba9ab42d04b42099e",
-            },
+            files={"form_input_name": ("filename", BytesIO(b"file content"))},
```

Comment:
the rationale for changing this and using files/data wargs instead of building by hand the multipart content is: well who does this in real life ?
now maybe im missing something and this was designed to show a multi-part form with those params passes, in which case I can rework it with the new form_input_name input field I added